components/item/SummaryInfo.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="card mb-3 ">
+    <header
+      v-if="titles"
+      class="card-heading px-4 pt-4"
+    >
+      <template
+        v-for="(heading, index) in titles"
+      >
+        <h1
+          v-if="index === 0"
+          :key="index"
+          :lang="heading.code"
+        >
+          {{ heading.value }}
+        </h1>
+        <p
+          v-else
+          :key="index"
+          :lang="heading.code"
+          class="font-weight-bold"
+        >
+          {{ heading.value }}
+        </p>
+      </template>
+    </header>
+    <div
+      v-if="description"
+      class="description px-4"
```

Comment:
Perhaps give the card a padding "p-4", instead of spacing classes on this description div and the header.